torch/_export/passes/lift_constants_pass.py

```diff
@@ -167,7 +167,10 @@ def lift_constants_pass(
     for node in gm.graph.nodes:
         if node.op == "get_attr":
             constant_val = _get_attr(gm, node.target)
-            if constant_val in lifted_objs:
+            if (
+                not isinstance(constant_val, torch.utils._pytree.TreeSpec)
```

Comment:
are these changes specific to the subclass constructor support?

My guess is no - we'll also need these if export ever wants to support custom ops that accept arbitrary pytree types. Maybe worth a test for that as well? (cc @zou3519 )

Reply:
This is basically because now we have some new attrs on the graph module (input specs for flat_apply) and export tries to lift some constants. I refactored the code little bit to make it easier to understand.